omdb-client/api/src/main/java/com/garret/movies/omdb/api/OmdbClient.java

```diff
@@ -0,0 +1,13 @@
+package com.garret.movies.omdb.api;
+
+import com.garret.movies.dao.entity.Movie;
+import com.garret.movies.dao.entity.ShortMovie;
+
+import java.util.List;
+
+public interface OmdbClient {
+
+    List<ShortMovie> searchMovies(String title);
+
+    Movie searchByImdbId(String imdbId);
```

Comment:
@Garret-vin And this is not cool :( Always split layers. You have API layer and persistance layer. So don't mix them.
You should not work with  DB models here, create own DTO's to work with API